.mockery.yaml

```diff
@@ -0,0 +1,20 @@
+with-expecter: false
+disable-version-string: true
+outpkg: "mocks"
+dir: "{{.InterfaceDir}}/mocks/"
+mockname: "{{.InterfaceName}}"
+filename: "{{.InterfaceName}}.go"
+packages:
+  github.com/jaegertracing/jaeger/pkg/es:
+    config:
+      all: true
+  github.com/jaegertracing/jaeger/storage/spanstore: 
+    config:
+      all: true
+  github.com/jaegertracing/jaeger/proto-gen/storage_v1: 
+    config:
+      all: true
+  github.com/jaegertracing/jaeger/pkg/es/client:
+    config:
+      all: true
+
```

Comment:
We seem to have a lot of other packages for which we probably just generated mocks manually once. Would be nice to include them in this config too (could be a separate PR):
```
$ find . -name mocks
./crossdock/services/mocks
./cmd/ingester/app/processor/mocks
./cmd/ingester/app/consumer/mocks
./cmd/collector/app/sanitizer/cache/mocks
./proto-gen/storage_v1/mocks
./plugin/sampling/leaderelection/mocks
./plugin/storage/grpc/mocks
./storage/mocks
./storage/metricsstore/mocks
./storage/dependencystore/mocks
./storage/spanstore/mocks
./storage/samplingstore/mocks
./pkg/kafka/mocks
./pkg/cassandra/mocks
./pkg/es/mocks
./pkg/es/client/mocks
./pkg/distributedlock/mocks
```

Reply:
I'll add it back to the ticket